spec/grape/api_spec.rb

```diff
@@ -689,7 +689,7 @@ class DummyFormatClass
         'example'
       end
       put '/example'
-      expect(last_response.headers['Content-Type']).to eql 'text/plain'
+      expect(last_response.headers[rack_versioned_headers[:content_type]]).to eql 'text/plain'
```

Comment:
Can we get rid of `rack_versioned_headers` and instead do `Grape::HTTP::Headers::CONTENT_TYPE`?